velox/docs/monthly-updates/may-2024.rst

```diff
@@ -0,0 +1,69 @@
+***************
+May 2024 Update
+***************
+
+Documentation
+=============
+
+* Publish blog post about `optimizing TRY and TRY_CAST <https://velox-lib.io/blog/optimize-try-more>`_
+
+Core Library
+============
+
+* Optimize TRY and TRY_CAST for cases when many rows fail.
+* Add support for LEFT SEMI FILTER and RIGHT SEMI FILTER merge joins.
+* Add support for DECIMAL input to aggregations over distinct values. :pr:`9850`
+* Add support for specifying overwrite behavior when registering simple scalar functions. :pr:`9158`
+* Fix LEFT merge join with extra filter. :pr:`9862`
+* Fix Nested Loop join with empty build and extra filter. :pr:`9892`
+
+Presto Functions
+================
+
+* Add :func:`at_timezone` function.
+* Add support for VARBINARY input to :func:`approx_distinct` aggregate function.
+* Add support for CAST(varchar AS timestamp with time zone).
+* Add support for DECIMAL inputs to :func:`min_by` and :func:`max_by` aggregate functions.
+* Fix :func:`arrays_overlap` function for empty arrays. :pr:`9922`
+* Fix :func:`map_top_n` function to break ties by comparing keys.
+* Fix CAST(IntervalDayTime AS Varchar) for negative intervals. :pr:`9871`
+* Fix :func:`from_base64` for inputs without padding. :pr:`8647`
+
+Spark Functions
+===============
+
+* Add :spark:func:`expm1`, :spark:func:`get`, :spark:func:`rint`, :spark:func:`shuffle`,
+  :spark:func:`soundex`, :func:`unix_seconds`, :spark:func:`width_bucket` functions.
+* Add support for complex type inputs to :spark:func:`hash` and :spark:func:`xxhash64` functions.
+* Fix CAST(tinyint/smallint/integer/bigint as varbinary). :pr:`9819`
+* Fix return type for :spark:func:`sum` aggregate function with REAL input. :pr:`9818`
+
+Performance and Correctness
+===========================
+
+* Add Fuzzers for TableWriter and RowNumber operators.
```

Comment:
Add two items in Performance
```
Optimize memory arbitration to avoid cross query interferences to reduce overall query execution time
Add cache expiration function to simple LRU cache for remote io throttling support
```

Reply:
Thank you. Added.